src/components/PWAInstructions/index.jsx

```diff
@@ -384,7 +346,8 @@ const PWAInstructions = (props) => {
                       <ToggleButton
                         value={item}
                         aria-label={item}
-                        classes={classes}
+                        //classes={classes}
+                        className={styles.toggleButton}
                         selected={device === item ? true : false}
```

Comment:
```suggestion
                        selected={device === item ? true : false}
```
not your code but fix it please